modules/identity-provider-provisioning-user-lookup-mapping.adoc

```diff
@@ -1,35 +1,37 @@
 // Module included in the following assemblies:
 //
-// * orphaned
+// * authentication/identity_providers/configuring-basic-authentication-identity-provider.adoc
 
 [id="identity-provider-provisioning-user-lookup-mapping_{context}"]
 = Manually provisioning a user when using the lookup mapping method
 
 When using the `lookup` mapping method, user provisioning is done by an external system, via the API.
```

Comment:
I am not sure of what `done by an external system, via the API` exactly means. Maybe you can safely remove the whole `When using the ``lookup`` mapping method, user provisioning is done by an external system, via the API.` sentence.